### PR TITLE
Add .jsproj extension

### PR DESCRIPTION
**Pull request by thejameskyle**
https://github.com/search?utf8=✓&q=jsproj+extension:jsproj&type=Code&ref=searchresults=

Sample: https://github.com/MicrosoftEdge/JSBrowser/blob/master/JSBrowser/JSBrowser.jsproj

### CODE REVIEW

File: samples/JSProj/JSBrowser.jsproj [hunk 1]
```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (pchaigno) [inline samples/JSProj/JSBrowser.jsproj]: This file should be in `samples/XML/` ;)

Comment (thejameskyle) [inline samples/JSProj/JSBrowser.jsproj]: done

**Comment by pchaigno** [comment]
:+1: 

For reviewers: A [search on github.com](https://github.com/search?utf8=%E2%9C%93&q=xml+extension%3Ajsproj&type=Code&ref=searchresults) showing more examples.

**Comment by arfon** [comment]
:+1: thanks @thejameskyle. This will be live on GitHub later this week when we cut a new version of the Linguist gem.

**Pull request merged**